document/CoreFormat.mdk

```diff
@@ -2281,6 +2287,17 @@ Tools that do not construct `fullDescription` in this way should provide a value
 because otherwise, the initial portion of `fullDescription` that a viewer displays where available space is limited
 might not be understandable.
 
+### `defaultLevel` property { #rule-defaultLevel }
+
+A `rule` object may contain a property named `defaultLevel` whose value shall be one of
+the strings defined for the `result.level` property (&sect;[#result-level]).
```

Comment:
If I just say it this way, `"notApplicable"` is allowed. Should I mention the allowed values explicitly?